From d55ce1608d03fcdf31cd7c5a19e28dc2535b5929 Mon Sep 17 00:00:00 2001
From: Ignacio Casal Quinteiro <icq@gnome.org>
Date: Tue, 15 Mar 2016 00:07:15 +0100
Subject: [PATCH 1/1] Bump libgit2-glib to 0.23.11 and fix api break

---
 libgitg/gitg-remote.vala | 2 +-
 1 file changed, 2 insertions(+), 0 deletions(-)

diff --git a/libgitg/gitg-remote.vala b/libgitg/gitg-remote.vala
index 991d4cf..6637312 100644
--- a/libgitg/gitg-remote.vala
+++ b/libgitg/gitg-remote.vala
@@ -247,7 +247,7 @@ public class Remote : Ggit.Remote
 			try
 			{
 				yield Async.thread(() => {
-					base.connect(direction, new Callbacks(this, callbacks, null));
+					base.connect(direction, new Callbacks(this, callbacks, null), null);
 				});
 			}
 			catch (Error e)
-- 
2.9.0

